lib/rules/avoid-lodash.js

```diff
@@ -24,12 +24,13 @@ module.exports = {
             description: "disallow importing lodash",
             category: "Code improvements",
             recommended: false,
-            url: "https://eslint.org/docs/rules/avoid-lodash"
+            url: "https://www.ec0lint.com/features/avoid-lodash",
```

Comment:
it should use only http (without www)

Reply:
+all other rules

Reply:
fix url for all other rules and add ec0lint's url to other rules which will have docs in the future